patch/kernel/archive/rockchip-rk3588-6.11/0026-RK3588-Add-VPU121-H.264-Decoder-Support.patch

```diff
@@ -309,3 +348,33 @@ index 111111111111..222222222222 100644
 -- 
 Armbian
 
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Muhammed Efe Cetin <efectn@protonmail.com>
+Date: Mon, 29 Jul 2024 23:21:19 +0300
+Subject: arm64: dts: rockchip: rk3588: disable H264 decoding on Hantro decoder
+
+---
+ drivers/media/platform/verisilicon/rockchip_vpu_hw.c | 7 +++----
+ 1 file changed, 3 insertions(+), 4 deletions(-)
+
+diff --git a/drivers/media/platform/verisilicon/rockchip_vpu_hw.c b/drivers/media/platform/verisilicon/rockchip_vpu_hw.c
+index 111111111111..222222222222 100644
+--- a/drivers/media/platform/verisilicon/rockchip_vpu_hw.c
++++ b/drivers/media/platform/verisilicon/rockchip_vpu_hw.c
+@@ -768,10 +768,9 @@ const struct hantro_variant rk3568_vepu_variant = {
+ 
+ const struct hantro_variant rk3568_vpu_variant = {
+ 	.dec_offset = 0x400,
+-	.dec_fmts = rockchip_vdpu2_dec_fmts,
+-	.num_dec_fmts = ARRAY_SIZE(rockchip_vdpu2_dec_fmts),
+-	.codec = HANTRO_MPEG2_DECODER |
+-		 HANTRO_VP8_DECODER | HANTRO_H264_DECODER,
++	.dec_fmts = rk3399_vpu_dec_fmts,
++	.num_dec_fmts = ARRAY_SIZE(rk3399_vpu_dec_fmts),
++	.codec = HANTRO_MPEG2_DECODER | HANTRO_VP8_DECODER,
```

Comment:
After disabling H264 decoder, rk3568_vpu_variant is same as rk3328, so rk3568 compatible string in no longer necessary in hantro driver. I have deleted it in rockchip64 family: https://github.com/armbian/build/blob/main/patch/kernel/archive/rockchip64-6.10/media-0003-rk3568-disable-hantro-h264.patch